content/en/docs/reference/glossary/cluster.md

```diff
@@ -4,14 +4,14 @@ id: cluster
 date: 2019-06-15
 full_link: 
 short_description: >
-   A set of machines, called nodes, that run containerized applications managed by Kubernetes. A cluster has at least one worker node and at least one master node. 
+   A set of worker machines, called nodes, that run containerized applications. Every cluster has at least one worker node.
 
 aka: 
 tags:
 - fundamental
 - operation
 ---
-A set of machines, called nodes, that run containerized applications managed by Kubernetes. A cluster has at least one worker node and at least one master node. 
+A set of worker machines, called nodes, that run containerized applications. Every cluster has at least one worker node.
 
 <!--more-->
-The worker node(s) host the pods that are the components of the application. The master node(s) manages the worker nodes and the pods in the cluster. Multiple master nodes are used to provide a cluster with failover and high availability.
+The worker node(s) host the pods that are the components of the application. The Control Plane manages the worker nodes and the pods in the cluster. In production environments, it's typical to run a Control Plane across multiple computers and also to run multiple nodes, to provide fault-tolerance and high availability.
```

Comment:
```suggestion
The worker node(s) host the pods that are the components of the application. The Control Plane manages the worker nodes and the pods in the cluster. In production environments, a Control Plane usually runs across multiple computers and a cluster usually runs multiple nodes, providing fault-tolerance and high availability.
```

Mostly grammar – if it significantly changes the meaning, let's talk about it :)